easybuild/easyconfigs/n/NAMD/NAMD-2.14_Linux-POWER-cuda.patch

```diff
@@ -0,0 +1,22 @@
+--- arch/Linux-POWER.cuda.orig	2020-08-05 18:51:20.000000000 +0100
```

Comment:
@edmondac please add author and short description to the top of the patches?

Reply:
Done